src/app/task/create-meeting/create-meeting.component.ts

```diff
@@ -60,6 +64,11 @@ export class CreateMeetingComponent implements OnInit {
           this.membersList = members.content;
           this.setAssignedMembers();
         });
+
+        this.groupService.loadGroupDetailsFromServer(this.groupUid).subscribe(resp => {
+          this.isGroupPaidFor = resp.paidFor;
+          console.log("Is this group paid for???",this.isGroupPaidFor);
+        });
```

Comment:
Clean this up. Also, should be able to get this in a more efficient way than a whole server call.